pkg/operator/controller.go

```diff
@@ -40,6 +42,9 @@ const (
 	kubeSystemNamespace      = "kube-system"
 	workqueueKey             = "changes"
 	defaultResyncDuration    = 10 * time.Minute
+	caBundlePath             = "/etc/pki/ca-trust/source"
+	caBundleFileName         = "ca-bundle.crt"
```

Comment:
need to discuss/confirm with @JacobTanenbaum but i think what his controller will be injecting into the configmap will be a .pem encoded file.  

What I had in my head is something that we could just mount over top of `/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem`

As that file is pointed to by the `/etc/pki/tls/certs/ca-bundle.crt` symlink anyway, and it's better to change the actual file in case anything is reading it from there, rather than just stomping the symlink.

we may *also* need to mount it to `/usr/share/pki/ca-trust-legacy/ca-bundle.legacy.default.crt` since that file is symlinked from `/etc/pki/ca-trust/source/ca-bundle.legacy.crt`  (ie another location some libraries may read from)

Reply:
@bparees We currently have the following content on ` /etc/pki/ca-trust/extracted/pem/` inside the `cluster-image-registry-operator` container:

```
sh-4.2$ ls -1 /etc/pki/ca-trust/extracted/pem/
README
email-ca-bundle.pem
objsign-ca-bundle.pem
tls-ca-bundle.pem
sh-4.2$
```

If we mount the "trusted ca config map" on `/etc/pki/ca-trust/extracted/pem/` we will loose some of these files, right? I remember @adambkaplan mentioned something about this but I can't remember what was it.

Reply:
@ricardomaraschini yes, if we mount here with no content, the files get wiped out. That's a feature per @smarterclayton - if the cluster hasn't determined who to trust, trust no one.

Reply:
I am resolving this one then :-)